googleanalytics.Rmd

```diff
@@ -24,16 +22,69 @@ ga_link_clicks <- readr::read_tsv(file.path("metricminer_data", "ga", "ga_link_c
+  * `sessions`: The number of sessions that began on your website or application. A session is a period of time during which a user interacts with your website or app. A session initiates when a user either opens your app in the foreground or views a page or screen and no session is currently active (e.g., their previous session has timed out). By default, a session ends (times out) after 30 minutes of user inactivity. There is no limit to how long a session can last.
+  * `averageSessionDuration`: The average duration (in seconds) of users' sessions. 
+  * `screenPageViews`: The number of mobile app screens or web pages your users saw. Repeated views of a single screen or page are counted. 
+  * `engagementRate`: The percentage of sessions that were engaged sessions. Specifically, the number of engaged sessions divided by the total number of sessions. An engaged session is a session that lasts 10 seconds or longer or has 1 or more conversion events or 2 or more page or screen views. (A conversion event is any user action that's valuable to your business and is defined by marking the event as a conversion. This is in contrast to most other metrics that are populated automatically.)  
+
 ```{r}
 knitr::kable(ga_metrics)
 ```
 
+**In the preview section above**: *Add plots that are like Figures 9 and 10 from the ITN evaluation*
+
+Data information for google analytics dimensions (`ga_dimensions`):
+
+  * `website`: Indicates the name of the website associated with that session. 
+  * `day`: Indicates the day of the session
+  * `month`: Indicates the month of the session
+  * `year`: Indicates the year of the session
+  * `country`: Indicates the country from which a session originates
+  * `fullPageURL`: Indicates full page URL of the page that is viewed in that session. 
+
 
 ```{r}
 knitr::kable(ga_dimensions)
 ```
 
+**In the preview section above**: *Add a plot that shows a map and distribution of countries? (though this might not generalize well)* or *Add a plot that shows traffic over time*
+
+Data information for google analytics link clicks (`ga_link_clicks`):
+
+  * `website`: Indicates the website on which the outbound click event occurred
+  * `linkURL`: Indicates the URL for the outbound click. What does NA mean for this -- a random click on the website?
+
 
 ```{r}
 knitr::kable(ga_link_clicks)
 ```
+
+**In the preview section above**: *Add a plot that shows the most visited linkURL* or *Add a plot that shows how many outbound clicks are associated with each website*
+
+## Setting up Google Analytics
+
+Grab information from vignette
+
+## Customization
+
+### Customizing Google Analytics Data
+
+Google Analytics explains that each dimension and metric that can be collected and reported has a scope (user-, session-, or hit-level) and in most cases, only dimensions and metrics that share the same scope can be combined. 
```

Comment:
Sounds good! We can link them to https://analytics.google.com/analytics/academy/ to learn more and that they can go to their own dashboard at https://analytics.google.com/analytics/ for how they can customize and explore what's possible to collect!

Reply:
Thanks! I'll add these links